### PR TITLE
Naz user profile

### PR DESCRIPTION
**Pull request by sarvmr**

### CODE REVIEW

**Comment by sarvmr** [comment]
user profile, without database connection

File: views/userProfile.html [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+                                        <circle cx="12" cy="12" r="10"></circle>
+                                        <line x1="2" y1="12" x2="22" y2="12"></line>
+                                        <path
+                                            d="M12 2a15.3 15.3 0 0 1 4 10 15.3 15.3 0 0 1-4 10 15.3 15.3 0 0 1-4-10 15.3 15.3 0 0 1 4-10z">
+                                        </path>
+                                    </svg>Website</h6>
+                                <span class="text-secondary">https://fromdatabase.com</span>
+                            </li>
+                            <!-- <li class="list-group-item d-flex justify-content-between align-items-center flex-wrap">
+                                <h6 class="mb-0"><svg xmlns="http://www.w3.org/2000/svg" width="24" height="24"
```
Comment (Rai-Sahil) [inline views/userProfile.html]: comments

File: views/userProfile.html [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+                                    <h6 class="mb-0">Phone</h6>
+                                </div>
+                                <div class="col-sm-9 text-secondary">
+                                    (239) 816-9029
+                                </div>
+                            </div>
+                            <!-- <hr> -->
+                            <!-- <div class="row">
+                                <div class="col-sm-3">
+                                    <h6 class="mb-0">Mobile</h6>
```
Comment (Rai-Sahil) [inline views/userProfile.html]: comments

**Review by Rai-Sahil — CHANGES_REQUESTED** [review]
Can you remove some of the commented code

**Review by Rai-Sahil — APPROVED** [review]
LGTM!!

**Pull request merged**